src/geoserver-create-imagemosaic-datastore/index.js

```diff
@@ -31,19 +31,22 @@ const grc = new GeoServerRestClient(url, user, pw);
  * @param {Array} inputs The inputs for this process
  *   First input is the workspace to publish to
  *   Second input is the name of the coverage store to be created
+ *   Third input is the path to prototype dataset for the coverage to be supported
  * @example
     {
        "id": 123,
        "type": "geoserver-create-imagemosaic-datastore",
        "inputs": [
            "klips",
-           "my-coverageStore"
+           "my-coverageStore",
+           "/path/to/tif/20220909T1400.tif"
         ]
     }
  */
 const geoserverCreateImageMosaicDatastore = async (workerJob, inputs) => {
   const ws = inputs[0];
   const covStore = inputs[1];
+  const prototypeGranule = inputs[2];
```

Comment:
Right, for our scenario it should be okay. But to make this worker more general, the creation of an empty store should be possible. I think this would also apply additions in the `geoserver-node-client`.